code/__DEFINES/flags.dm

```diff
@@ -63,6 +63,7 @@
 #define NOGUNS			512
 #define NOBLOOD			1024
 #define NOFIRE			2048
+#define NOHUNGER		2049

```

Comment:
You actually can't use 65536, it's too big for a define. Even though these aren't bitflags, that bit is still relevant.

Rebase and 
`#define NOHUNGER		17`